Makefile

```
@@ -65,39 +65,55 @@ vpath %.c $(SRCD)
 all : $(NAME)
 
 $(NAME):	$(OBJS) $(LIBEXT)
```

1. Compile all libraries
2. Compile our sources codes
```suggestion
$(NAME):	$(LIBEXT) $(OBJS)
```